civiform/civiform
# Update address correction page to use new strings

### Description

This PR updates the text on the address correction page to use the new strings specified by Sijia.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Created unit and/or browser tests which fail without the change (if possible) -- **existing ones are sufficient**

#### User visible changes

  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag) -- **Strings have been translated**

### Instructions for manual testing

- Fill out an address question that has address correction labeled with "Legit Address" as the street name. On the address correction page, verify the new strings are shown and the suggestions are shown.
- Fill out an address question that has address correction labeled with "Bogus Address" as the street name. On the address correction page, verify the new strings are shown and no suggestions are shown.

### Issue(s) this completes

Fixes #6639
